File: snippets/api/getting-started/fabric.java
package com.plasmovoice.fabricmod;

import su.plo.voice.api.server.PlasmoVoiceServer;

import net.fabricmc.api.ModInitializer;

public final class FabricMod implements ModInitializer {

    // Addon class annotated with @Addon.
    private final TestAddon addon = new TestAddon();

    @Override
    public void onInitialize() {
        PlasmoVoiceServer.getAddonsLoader().load(addon);
    }
}

File: snippets/api/getting-started/fabric.kt
package com.plasmovoice.fabricmod

import su.plo.voice.api.server.PlasmoVoiceServer

import net.fabricmc.api.ModInitializer

class FabricMod : ModInitializer {

    // Addon class annotated with @Addon.
    private val addon = TestAddon()

    override fun onInitialize() {
        PlasmoVoiceServer.getAddonsLoader().load(addon)
    }
}
